tests/integration/security/sds_ingress/util/util.go

```diff
@@ -162,8 +164,9 @@ func createSecret(ingressType ingress.CallType, cn, ns string, ic IngressCredent
 			Namespace: ns,
 		},
 		Data: map[string][]byte{
-			tlsScrtCert: []byte(ic.ServerCert),
-			tlsScrtKey:  []byte(ic.PrivateKey),
+			tlsScrtCert:   []byte(ic.ServerCert),
+			tlsScrtKey:    []byte(ic.PrivateKey),
+			tlsScrtCaCert: []byte(ic.CaCert),
```

Comment:
This Doesn't seem right. `ingressType != ingress.Mtls` yet we are creating a mTLS secret?

For plain TLS we should not have a ca cert